src/Tutorials/Basics/HDA_flowsheet_solution_testing.ipynb

```diff
@@ -1003,7 +1003,7 @@
     "solver.options = {'tol': 1e-6, 'max_iter': 5000}\n",
```

Comment:
We should replace this with the `get_solver` method, and set the solver options through it.